ceph-pr-commits/build/build

```diff
@@ -1,9 +1,25 @@
 #!/bin/bash
 
+# Don't require signed commits if only docs changed.
+# Modified from ceph-docs/build/build.
+# I tried using the excluded-regions parameter for the ghprb plugin but since
+# this job/check is required, it hung with 'Expected — Waiting for status to be reported'
+cd "$WORKSPACE/ceph"
+if git rev-parse --verify ${GIT_COMMIT}^2; then
```

Comment:
```
+ pushd .
~/build/workspace/ceph-pr-commits ~/build/workspace/ceph-pr-commits
+ cd /home/jenkins-build/build/workspace/ceph-pr-commits/ceph
+ git rev-parse --verify 'e9dd427c6dec2b6282d4e2643f79df1eb639f256^2'
fatal: Needed a single revision
```

@djgalloway @jdurgin i spotted this failure in https://jenkins.ceph.com/job/ceph-pr-commits/33940/console .

any idea?

Reply:
Yeah, I could probably use @dmick's expertise here.  When I was researching the best ways to just "list what files changed in a PR," `git rev-parse ${GIT_COMMIT}^2` was what came up.  Maybe we can get rid of `--verify`?

```
       --verify
           Verify that exactly one parameter is provided, and that it can be turned into a raw 20-byte SHA-1 that can be used to access the object database. If so, emit it to the standard output; otherwise, error out.

           If you want to make sure that the output actually names an object in your object database and/or can be used as a specific type of object you require, you can add the ^{type} peeling operator to the parameter. For example, git rev-parse "$VAR^{commit}"
           will make sure $VAR names an existing object that is a commit-ish (i.e. a commit, or an annotated tag that points at a commit). To make sure that $VAR names an existing object of any type, git rev-parse "$VAR^{object}" can be used.
```

(That job failed because of a legit commit problem though, right?)

Reply:
@djgalloway any update? there are more and more test failures caused by this issue. see also #36544